ykrt/src/trace/swt/mod.rs

```diff
@@ -111,20 +108,23 @@ impl TraceRecorder for SWTTraceRecorder {
                             "Failed to get function name by index {:?}",
                             tb.function_index
                         ),
-                    })
-                    .collect();
+                    }
+                }));
             })
         });
-        Ok(Box::new(SWTraceIterator::new(aot_blocks)))
+        if aot_blocks.len() > DEFAULT_TRACE_TOO_LONG {
```

Comment:
What about this stuff? Looks like some semantic fixes.

That's fine, but I was expecting only fixing warnings :)

Reply:
It's both. The warning was correct: the backend *has* to handle "trace too long" and it wasn't referencing the `DEFAULT_TRACE_TOO_LONG` variable.

Reply:
I had to look this one up to remind myself!